clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/IsolatedIngestionBackend.java

```diff
@@ -153,10 +152,11 @@ public void dropStoragePartitionGracefully(
           // Clean up the topic partition ingestion status.
           mainIngestionRequestClient.resetTopicPartition(topicName, partition);
         });
+    mainIngestionMonitorService.cleanupTopicPartitionState(topicName, partition);
```

Comment:
Why does the ordering of these statements matter? I don't have any concern, but am just curious.

Reply:
With the final fix,  it does not matter. This change happens during initial debugging, but I think either place should be fine.

Reply:
If it does not matter then I would lean towards not changing it. This way the git history is easier to understand. But it's up to you.